test/extensions/filters/http/jwt_authn/authenticator_test.cc

```diff
@@ -94,104 +97,110 @@ TEST_F(AuthenticatorTest, TestForwardJwt) {
   // Test OK pubkey and its cache
   auto headers = Http::TestHeaderMapImpl{{"Authorization", "Bearer " + std::string(GoodToken)}};
 
-  MockAuthenticatorCallbacks mock_cb;
-  EXPECT_CALL(mock_cb, onComplete(_)).WillOnce(Invoke([](const Status& status) {
+  std::function<void(const Status&)> on_complete_cb = [](const Status& status) {
```

Comment:
Seems this pattern appeared many times in this file, can you extract this to a helper function, like expectVerifyStatus(Status status) in AuthenticatorTest?

```
  std::function<void(const Status&)> on_complete_cb = [](const Status& status) {
    ASSERT_EQ(status, Status::JwtAudienceNotAllowed);
  };
  auto tokens = filter_config_->getExtractor().extract(headers);
  auth_->verify(headers, std::move(tokens), std::move(on_complete_cb));
```

Reply:
moved to expectVerifyStatus method.